### PR TITLE
Newbranch

### PR DESCRIPTION
**Pull request by igorkol91**
NYT article clone
fixed all the mandatory fixes asked in the HTML and css
edited readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+New York Times Article Clone
```
Comment (christianotieno) [inline README.md:1]: ## Improve the README
- [ ] Inline with having a descriptive README, do you mind adding an alive link to where the page is hosted live? 

You can use this [template ](https://github.com/microverseinc/readme-template)from [pathwright ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)for inspiration. You are also welcomed to use this [tool ](https://raw.githack.com/)to generate a live link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
```
Comment (christianotieno) [inline index.html:1]: ### Validate the markup
- [ ] We need to ensure that our markup does not have any informing warnings. Do you mind validating our markup?
You are free to use this [tool ](https://validator.w3.org/nu/#textarea)for easier validation.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+<body>
+    <header class="flex">
+        <div id="left-header" class="inline-flex">
+            <button><img src="./hamburger-menu-lines-icon.png" alt="burger-menu"></button>
+            <button id="search-button"><img src="./index.png" alt="search-button"></button>
+            <p id="hidden-p">SPACE & COSMOS</p>
+        </div>
+
+        <div id="mid-header">
+            <img src="./ny-times-logo.png" alt="main-header-logo">
```
Comment (christianotieno) [inline index.html]: ### Navigation bar
- [ ] Kindly ensure The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)). Currently, we have an image which is a good thing. Do you mind turning it into a link?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+                            <figcaption>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded
+                                <br> so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit.<span class="credit">Rick Friedman for The New York</span> Times</figcaption>
+                        </figure>
+                    </div>
+                </div>
+            </section>
+            <section id="section-2">
+                <div>
+                    <div class="new-line"></div>
+
```
Comment (christianotieno) [inline index.html]: ### ADD missing sections in the main article.
- [ ] There are some missing social links at the top of the main article's image. Do you mind adding them?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/igorkol91/NYTarticle/newbranch/index.html)|
|:--:|:--:|
|Main-Article-socials-source|Main-Article-socials|

File: index.html [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+                    <div class="new-line"></div> The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.
+                    <div class="new-line"></div> Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday
+                    and in a set of papers submitted to The Astrophysical Journal.
+                    <div class="new-line"></div>
+                    <div id="inflation-img" class="flex">
+                        <h4>The Theory of Inflation</h4>
+                        <p>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+                        <img src="./inflation-945.png" alt="inflation-big-img">
```
Comment (christianotieno) [inline index.html]: ### Required: Full implementation of the `Theory of Inflation`
- [ ] This section you appeared to have to the screenshot and included it in the clone page. Kindly ensure that this section is fully implemented and all the items styled accordingly. You stand a chance to gain more experience from its implementation. Avoid using screenshots of the original page as a shortcut to bypass requirements for this and future projects. Thank you.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+                    who is now a professor at Massachusetts Institute of Technology (as is his son, Larry, who was sleeping that night in 1979), in his office last week.
+                    <div class="new-line"></div> “It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes.
+                    <div class="new-line"></div> Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford
+                    colleague and a member of Dr. Kovac’s team, knocked on his door with a bottle of Champagne to tell him the news.
+                    <div class="new-line"></div>
+                    <iframe width="420" height="315" title="youtube-vid" src="https://youtu.be/ZlfIVEy_YOA">
```
Comment (christianotieno) [inline index.html]: ### FIX the video not showing
- [ ] The video currently is not showing as expected. Do you mind fixing this?

Below is a comparison of what is expected and what we currently have.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/igorkol91/NYTarticle/newbranch/index.html)|
|:--:|:--:|
|video|video-clone|

File: index.html [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+                        view.
+                        <div class="new-line"></div> “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+                        <div class="new-line"></div>
+                </div>
+
+            </section>
+        </article>
+
+
+
```
Comment (christianotieno) [inline index.html]: ### Implement missing sections 
The following sections have not been implemented on the clone page. I have highlighted them below for easier reference. 

#### Suggested newsletters section

**NOTE: Do you mind also include in the implementation, the `READ 615 Comments` and the preceding social links as well?**

Suggested-newsletters-section

#### More into space and astronomy

more-inspace-and-astronomy

#### Editors pick

editors-pick-footer

File: index.html [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+                <ul>
+                    <li>Arts</li>
+                    <li><a href="#" class="footer-link">Today's Arts</a></li>
+                    <li><a href="#" class="footer-link">Art & Design</a></li>
+                    <li><a href="#" class="footer-link">Books</a></li>
+                    <li><a href="#" class="footer-link">Dance</a></li>
+                    <li><a href="#" class="footer-link">Movies</a></li>
+                    <li><a href="#" class="footer-link">Music</a></li>
+                    <li><a href="#" class="footer-link">Pop Culture</a></li>
+                    <li><a href="#" class="footer-link">Television</a></li>
```
Comment (christianotieno) [inline index.html]: ### Imrove the `Footer` Section

- [ ] A closer look at the footer, we can see the contrast. On the Source page, we have the page's logo and contents are divided into visible borders. Do you mind improving these areas?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/igorkol91/NYTarticle/newbranch/index.html)|
|:--:|:--:|
|footer|footer-clone|

File: index.html [hunk 9]
```diff
@@ -0,0 +1,316 @@
...
+            <li>T Brand Studio</li>
+            <li>Your Ad Choices</li>
+            <li>Privacy Policy</li>
+            <li>Terms of Service</li>
+            <li>Terms of Sale</li>
+            <li>Site Map</li>
+            <li>Help</li>
+            <li> Subscriptions</li>
+        </ul>
+    </section>
```
Comment (christianotieno) [inline index.html:532]: ### Optional Change
- [ ] Do you mind while conducting the above changes, implement the sticky section bottom of the footer that sticks to the bottom of the page as well?

File: main.css [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+
+footer li:first-child {
+  font-weight: bold;
+  margin-bottom: 10px;
+}
+
+.top-subscribe {
+  font-weight: bold;
+  margin-bottom: 10px;
+}
```
Comment (christianotieno) [inline main.css:327]: ## Aim to remain DRY
- [ ] We appear to be repeating ourselves in lines `#193` & `#198`. Do you mind merging these rules?

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @igorkol91 👋🏾 

Thank you for submitting your `Positioning and Floating Elements` project for the first review. The codebase is clean, DRY for the majority of parts of the codebase saved for a few, and free of commented out code. It is quite commendable how you have strived to keep the repository professional for majority of parts.  Keep it up 👏🏾 

Before this gets approved, we need to make sure nothing is messed up. Kindly take care of some highlighted issues down below. 👇🏾 

**Kindly note: that this will act as a first review and after which the pull request will have only two more review chances for it to get approved. I would urge we try and work on the suggested changes with utmost keenness** 

### Mandatory changes
- [ ] Improve the `navbar`  to match the source page.
- [ ] Improve the main article's image section to match the source page.
- [ ] Improve the `Theory of Inflation' section to match the source page.
- [ ] Improve the footer of the page.
- [ ] Implement the highlighted missing sections.
- [ ] Improve the project's highlighted sections.
### Happy coding 💻 
#### Christian Otieno (CR)

File: README.md (hunk 1 above)
Comment (christianotieno) [reply README.md:1]: ### Improve the Commit and Pull Request Message
- [ ] Your commit messages don't appear to be well informed of what the changes getting added are dong. Do you mind being more elaborate with the commit messages?
- [ ] Kindly note that the Pull Request message ought to communicate what the changes in the said branch are introducing in the main codebase (Changes to be merged). So your mind improving the Pull Request message to showcase to the reviewer what changes/work was carried out in this branch? 

Check this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) for inspiration on how to go about it.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @igorkol91 :wave:

__You've really done well so far. Thank you for fixing most of the required changes by the last reviewer:__

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] The live demo link is still missing in your Readme. Are you having any challenge(s) doing this? You can use either Github pages or the [githack cdn](https://raw.githack.com/)
- [ ] As mentioned by the last reviewer, you're having some formatting issues in your Readme. Kindly use [this template](https://github.com/microverseinc/readme-template) as a guide.
- [ ] Looks like the newly implemented sections are not properly structured. Kindly check the `Suggested news for you`, `More in Space and Astronomy`, `Most Popular`, and the bottom navigation items. Please compare your implementation with that of the original page and make all necessary corrections.
- [ ] You're having the `.vscode` directory in your remote folder structure. Do not commit any file/folder that is not related to the project. Let's fix this. 
The easiest way to do this is to:
- Delete the `.vscode` locally from your project root directory
- Add the `.vscode` directory to your `.gitignore` file. The [.gitignore](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/) is a file, not a folder (I can see you added it as a folder). Do not skip the initial `.`, which is part of the directory name (i.e. `/.vscode`).
- Add and commit your changes.
- You can then restore the `.vscode` directory from the Cycle bin to your project if you still need it for developmental purposes.

**[ OPTIONAL CHANGES ]**

- [ ] You've used a lot of `id` selectors for your CSS rules. Personally, I feel like this behavior is not the conventional way of doing things. You should use the `class` selectors extensively in your project. This is because the `id` attribute has higher precedence than its `class` counterpart. If care is not taken, it might end up messing with the cascading nature of your CSS rules. You may want to refactor your code.

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second-opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by christianotieno — COMMENTED** [review]
## Project Approved 🍾 
Hello @igorkol91 👋🏾 
Congratulations. The project is now approved. 👏🏾 
All the very best in your next tasks 🥇 

Happy coding 💻 
Christian Otieno (CR)

**Pull request merged**